joern-cli/frontends/csharpsrc2cpg/src/test/scala/io/joern/csharpsrc2cpg/passes/DependencyTests.scala

```diff
@@ -141,4 +141,28 @@ class DependencyTests extends CSharpCode2CpgFixture {
 
   }
 
+  "a csproj file specifying a dependency with trailing or leading whitespaces" should {
+    val config = Config().withDownloadDependencies(true);
+    "not throw a exception" in {
+      val cpg = code("""
+          |namespace Foo;
+          |""".stripMargin)
+        .moreCode(
+          """
+            |<Project Sdk="Microsoft.NET.Sdk">
+            |    <ItemGroup>
+            |      <PackageReference Include=" System.Security.Cryptography.Pkcs" Version="6.0.4"/>
```

Comment:
Is it possible for the `Version` to also have leading or trailing spaces?

Reply:
It's a bug we found on one repository, both of them shouldn't have spaces.